hb.elpi

```diff
@@ -1030,7 +1030,7 @@ declare-instances T [class Class Struct MLwP|Rest] :-
   std.assert-ok! (coq.typecheck S STy) "declare-instances: S illtyped",
 
   coq.env.add-const Name S STy @transparent! CS, % Bug coq/coq#11155, could be a Let
-  coq.CS.declare-instance (const CS), % Bug coq/coq#11155, should be local
+  with-locality (coq.CS.declare-instance (const CS)), % Bug coq/coq#11155, should be local
```

Comment:
@CohenCyril @pi8027 : the comment here seems to suggest we want local-by-default.
If so, the attribute should be `#[global]` instead...